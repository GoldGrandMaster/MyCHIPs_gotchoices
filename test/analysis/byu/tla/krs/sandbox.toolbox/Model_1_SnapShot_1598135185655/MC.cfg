\* SPECIFICATION definition
SPECIFICATION
Spec
\* Generated on Sat Aug 22 16:26:21 MDT 2020