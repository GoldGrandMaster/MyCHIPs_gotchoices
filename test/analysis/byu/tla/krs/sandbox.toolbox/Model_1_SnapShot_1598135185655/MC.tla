---- MODULE MC ----
EXTENDS sandbox, TLC

=============================================================================
\* Modification History
\* Created Sat Aug 22 16:26:21 MDT 2020 by kylestorey
