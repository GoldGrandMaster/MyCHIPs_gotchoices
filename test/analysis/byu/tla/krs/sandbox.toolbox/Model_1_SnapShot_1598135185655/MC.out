@!@!@STARTMSG 2262:0 @!@!@
TLC2 Version 2.15 of Day Month 20?? (rev: eb3ff99)
@!@!@ENDMSG 2262 @!@!@
@!@!@STARTMSG 2187:0 @!@!@
Running breadth-first search Model-Checking with fp 100 and seed -3757364789755925660 with 2 workers on 4 cores with 608MB heap and 1365MB offheap memory [pid: 64571] (Mac OS X 10.13.6 x86_64, AdoptOpenJDK 14.0.1 x86_64, OffHeapDiskFPSet, DiskStateQueue).
@!@!@ENDMSG 2187 @!@!@
@!@!@STARTMSG 2220:0 @!@!@
Starting SANY...
@!@!@ENDMSG 2220 @!@!@
Parsing file /Users/kylestorey/workspace/MyCHIPs/test/analysis/krs/sandbox.toolbox/Model_1/MC.tla
Parsing file /Users/kylestorey/workspace/MyCHIPs/test/analysis/krs/sandbox.toolbox/Model_1/sandbox.tla
Parsing file /Applications/TLA+ Toolbox.app/Contents/Eclipse/plugins/org.lamport.tlatools_1.0.0.202004251858/tla2sany/StandardModules/TLC.tla
Parsing file /Applications/TLA+ Toolbox.app/Contents/Eclipse/plugins/org.lamport.tlatools_1.0.0.202004251858/tla2sany/StandardModules/Naturals.tla
Parsing file /Applications/TLA+ Toolbox.app/Contents/Eclipse/plugins/org.lamport.tlatools_1.0.0.202004251858/tla2sany/StandardModules/Sequences.tla
Parsing file /Applications/TLA+ Toolbox.app/Contents/Eclipse/plugins/org.lamport.tlatools_1.0.0.202004251858/tla2sany/StandardModules/FiniteSets.tla
Semantic processing of module Naturals
Semantic processing of module Sequences
Semantic processing of module FiniteSets
Semantic processing of module TLC
Semantic processing of module sandbox
Semantic processing of module MC
@!@!@STARTMSG 2219:0 @!@!@
SANY finished.
@!@!@ENDMSG 2219 @!@!@
@!@!@STARTMSG 2185:0 @!@!@
Starting... (2020-08-22 16:26:22)
@!@!@ENDMSG 2185 @!@!@
@!@!@STARTMSG 2189:0 @!@!@
Computing initial states...
@!@!@ENDMSG 2189 @!@!@
@!@!@STARTMSG 2269:0 @!@!@
Computed 2 initial states...
@!@!@ENDMSG 2269 @!@!@
@!@!@STARTMSG 2269:0 @!@!@
Computed 4 initial states...
@!@!@ENDMSG 2269 @!@!@
@!@!@STARTMSG 2269:0 @!@!@
Computed 8 initial states...
@!@!@ENDMSG 2269 @!@!@
@!@!@STARTMSG 2269:0 @!@!@
Computed 16 initial states...
@!@!@ENDMSG 2269 @!@!@
@!@!@STARTMSG 2190:0 @!@!@
Finished computing initial states: 20 distinct states generated at 2020-08-22 16:26:24.
@!@!@ENDMSG 2190 @!@!@
@!@!@STARTMSG 2193:0 @!@!@
Model checking completed. No error has been found.
  Estimates of the probability that TLC did not check all reachable states
  because two distinct states had the same fingerprint:
  calculated (optimistic):  val = 8.7E-17
@!@!@ENDMSG 2193 @!@!@
@!@!@STARTMSG 2201:0 @!@!@
The coverage statistics at 2020-08-22 16:26:24
@!@!@ENDMSG 2201 @!@!@
@!@!@STARTMSG 2773:0 @!@!@
<Init line 21, col 1 to line 21, col 4 of module sandbox>: 20:20
@!@!@ENDMSG 2773 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 24, col 12 to line 24, col 26 of module sandbox: 1
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 25, col 12 to line 25, col 26 of module sandbox: 20
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2772:0 @!@!@
<Transfer line 27, col 1 to line 27, col 8 of module sandbox>: 20:20
@!@!@ENDMSG 2772 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 27, col 16 to line 27, col 30 of module sandbox: 100
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 27, col 16 to line 27, col 17 of module sandbox: 80
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 28, col 19 to line 28, col 40 of module sandbox: 20
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 29, col 24 to line 29, col 35 of module sandbox: 10
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 30, col 24 to line 30, col 35 of module sandbox: 10
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 31, col 16 to line 31, col 64 of module sandbox: 20
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2772:0 @!@!@
<A line 33, col 1 to line 33, col 1 of module sandbox>: 10:10
@!@!@ENDMSG 2772 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 33, col 9 to line 33, col 16 of module sandbox: 90
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 33, col 9 to line 33, col 10 of module sandbox: 80
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 34, col 9 to line 34, col 46 of module sandbox: 10
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 35, col 9 to line 35, col 17 of module sandbox: 10
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 36, col 9 to line 36, col 42 of module sandbox: 10
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2772:0 @!@!@
<B line 38, col 1 to line 38, col 1 of module sandbox>: 10:10
@!@!@ENDMSG 2772 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 38, col 9 to line 38, col 16 of module sandbox: 90
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 38, col 9 to line 38, col 10 of module sandbox: 80
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 39, col 9 to line 39, col 42 of module sandbox: 10
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 40, col 9 to line 40, col 17 of module sandbox: 10
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 41, col 9 to line 41, col 44 of module sandbox: 10
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2772:0 @!@!@
<C line 43, col 1 to line 43, col 1 of module sandbox>: 20:20
@!@!@ENDMSG 2772 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 43, col 9 to line 43, col 16 of module sandbox: 100
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 43, col 9 to line 43, col 10 of module sandbox: 80
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 44, col 9 to line 45, col 60 of module sandbox: 20
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 46, col 9 to line 46, col 20 of module sandbox: 20
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 47, col 9 to line 47, col 57 of module sandbox: 20
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2772:0 @!@!@
<Terminating line 50, col 1 to line 50, col 11 of module sandbox>: 0:20
@!@!@ENDMSG 2772 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 50, col 16 to line 50, col 26 of module sandbox: 100
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 50, col 16 to line 50, col 17 of module sandbox: 80
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 50, col 31 to line 50, col 44 of module sandbox: 20
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2202:0 @!@!@
End of statistics.
@!@!@ENDMSG 2202 @!@!@
@!@!@STARTMSG 2200:0 @!@!@
Progress(5) at 2020-08-22 16:26:24: 100 states generated (2,193 s/min), 80 distinct states found (1,755 ds/min), 0 states left on queue.
@!@!@ENDMSG 2200 @!@!@
@!@!@STARTMSG 2199:0 @!@!@
100 states generated, 80 distinct states found, 0 states left on queue.
@!@!@ENDMSG 2199 @!@!@
@!@!@STARTMSG 2194:0 @!@!@
The depth of the complete state graph search is 5.
@!@!@ENDMSG 2194 @!@!@
@!@!@STARTMSG 2268:0 @!@!@
The average outdegree of the complete state graph is 1 (minimum is 0, the maximum 1 and the 95th percentile is 1).
@!@!@ENDMSG 2268 @!@!@
@!@!@STARTMSG 2186:0 @!@!@
Finished in 2768ms at (2020-08-22 16:26:24)
@!@!@ENDMSG 2186 @!@!@
